### DISCUSSION TITLE
programmable text on site

### DISCUSSION THREAD

**Discussion by vasnt** [statelyai/xstate#3711] [General] [answered]
https://stately.ai/  site says "Mature project with 19k GitHub stars" 
Could it be linked/programmed to update automaticallly based on actual stars as currently it is above 22k+

**Comment by davidkpiano** [comment, marked as answer]
Thanks! We'll make this change shortly.

**Reply by vasnt** [reply]
Please advice me later which solution/tool  is used/implemented to resolve this issue? I am curious to know about.
